examples/chip-tool/args.gni

```diff
@@ -28,6 +28,9 @@ matter_enable_tracing_support = true
 matter_log_json_payload_hex = true
 matter_log_json_payload_decode_full = true
 
+chip_enable_additional_data_advertising = true
+chip_enable_rotating_device_id = true
```

Comment:
How does this make sense for chip-tool at all?

Reply:
I've enabled it for the chip-tool since it's a controller. I thought that it might make sense to enable rotating device ID in the server part and the client part, so CI could verify potential build breaks. To be honest I'm not sure whether there is any command in chip-tool which can read this ID, but if there is none, it could be added, so the functionality could be tested.

Reply:
I think my point is that this compile flag was supposed to control whether a commissionable device has a rotating device id.  chip-tool is not a commissionable device....

I guess someone started using this flag in `AdditionalDataPayloadGenerator::generateAdditionalDataPayload`, which is not about commissionables.  So now what this flag means is anyone's guess.

Reply:
> I think my point is that this compile flag was supposed to control whether a commissionable device has a rotating device id. chip-tool is not a commissionable device....

OK, I see. So, maybe I could fix the usage of that flag as well. So, one more question. Since this flag is only for commissionable devices, than the support for reading such ID can/shall be in chip-tool regardless of any flag?

And as for the chip-tool. Maybe my wording was not right. I do not want to enable generation of rotating ID in the chip-tool, but only a support for reading it and printing to user (in logs or via dedicated command).

Reply:
I've read the code more carefully and indeed you are right. These options do not make any sense for chip-tool. Many thanks for pointing that out!